manifests/ppa.pp

```diff
@@ -5,6 +5,8 @@
   $options        = $::apt::ppa_options,
   $package_name   = $::apt::ppa_package,
   $package_manage = false,
+  $proxy_host     = undef,
+  $proxy_port     = 8080,
```

Comment:
Ok, cool, just wanted to make sure I didn't miss something new :) I'll get this updated.